### PR TITLE
Change account to sub-account

### PR DESCRIPTION
**Pull request by Buckram123**
Did a small test and it turns out transfers was supported!
So I decided to add recursive acceptance of the ownership for better user experience
### Checklist

- [x] CI is green.
- [x] Changelog updated.

### CODE REVIEW

**Comment by CyberHoward** [comment]
And it registers the sub-account on the main account after the transfer is accepted? damn!

File: framework/contracts/account/manager/src/contract.rs [hunk 1]
```diff
@@ -206,6 +206,16 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> M
                         cw_ownable::Action::AcceptOwnership => update_governance(deps.storage)?,
                         _ => vec![],
```
Comment (CyberHoward) [inline framework/contracts/account/manager/src/contract.rs:207]: We should clear the cache state (proposed governance) when the `RenounceOwnership` variant is called.

File: framework/contracts/account/manager/src/contract.rs [hunk 2]
```diff
@@ -206,6 +206,16 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> M
...
                     };
+                    let mut info = info;
+                    // if new governance is sub-account trick cw-ownable to think it was called by proxy
```
Comment (CyberHoward) [inline framework/contracts/account/manager/src/contract.rs:210]: I am very confused by this, could you elaborate on why this is needed?

Comment (CyberHoward) [reply framework/contracts/account/manager/src/contract.rs:212]: Ahhhh I get it!! Okey haha, sorry

File: framework/contracts/account/manager/src/contract.rs [hunk 3]
```diff
@@ -197,7 +197,9 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> M
                     handle_sub_account_action(deps, info, action)
                 }
                 ExecuteMsg::Callback(CallbackMsg {}) => handle_callback(deps, env, info),
+                // Used to claim or renounce an ownership change.
                 ExecuteMsg::UpdateOwnership(action) => {
+                    // Disallow the user from using the TransferOwnership action.
                     let msgs = match action {
                         // Disallow the user from using the TransferOwnership action
                         cw_ownable::Action::TransferOwnership { .. } => {
```
Comment (AndrewPochapsky) [inline framework/contracts/account/manager/src/contract.rs:205]: Can't comment directly on the line but should we rename the `ShouldUseSetOwner` error variant now?

File: framework/contracts/account/manager/src/error.rs [hunk 4]
```diff
@@ -88,7 +88,7 @@ pub enum ManagerError {
     #[error("invalid configuration action, {}", error)]
     InvalidConfigAction { error: StdError },
 
-    #[error("Must use SetOwner to change owner")]
+    #[error("Must use ProposeOwner to change owner")]
     MustUseSetOwner {},
```
Comment (AndrewPochapsky) [inline framework/contracts/account/manager/src/error.rs:92]: Should we also rename the variant itself?

File: framework/contracts/account/manager/src/contract.rs (hunk 2 above)
Comment (Buckram123) [reply framework/contracts/account/manager/src/contract.rs]: > I am very confused by this, could you elaborate on why this is needed?

Our `execute_update_ownership` macro checks new owner against `info.sender`, and to support top-level admin claims we need to edit sender when it's called by the top-level admin :)

File: framework/contracts/account/manager/src/error.rs (hunk 4 above)
Comment (Buckram123) [reply framework/contracts/account/manager/src/error.rs:92]: Sure, updated!

File: framework/contracts/account/manager/src/contract.rs (hunk 2 above)
Comment (Buckram123) [reply framework/contracts/account/manager/src/contract.rs]: moved it into `update_governance` function as well, so pending owner can't renounce himself(which is okay but still minor bug)

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**